src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/PersonDateOfBirthMapper.java

```diff
@@ -17,6 +21,12 @@ public PersonDateOfBirth map(Parameters parameters) {
 
     private Instant getPersonDob(Parameters parameters) {
         Patient patient = ParametersExtension.extractPatient(parameters);
-        return patient.getBirthDate().toInstant();
+        var birthDateElement = patient.getBirthDateElement();
+        return LocalDateTime
+            .of(
+                birthDateElement.getYear(), birthDateElement.getMonth() + 1, birthDateElement.getDay(),
+                birthDateElement.getHour(), birthDateElement.getMinute(), birthDateElement.getSecond())
```

Comment:
Could you use LocalDate and not include the hour, minute, and second?

Reply:
I could but the "birthDateElement" has an option to have time as well. Should we trim that ?

Reply:
Ok, it's only date. Fixing